.github/workflows/ci-build-and-push.yml

```diff
@@ -0,0 +1,45 @@
+name: ci-build-and-push
+
+on:
+  push:
+    branches:
+      - main  
+  pull_request:
+    branches:
+      - main  
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Set up QEMU for multi-architecture builds
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          platforms: linux/amd64,linux/arm64/v8
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_AUTHTOK }}
+
+      - name: Build and push multi-architecture image
+        uses: docker/build-push-action@v6
```

Comment:
`examples/multiubuntu/build/build.sh` can handle build and push as well, no need to setup docker action for it.
build and push

Reply:
We need multiarch builds, they are not handled in the script.

Reply:
my bad, in that case keep repo name same that used in the script

Reply:
> `examples/multiubuntu/build/build.sh` can handle build and push as well, no need to setup docker action for it. build and push

Do I need to make any changes? Do I need to use examples/multiubuntu/build/build.sh? A bit confused here. Would love for some clarity :))

Reply:
@sikehish no need to use build.sh script, changes with your PR looks good just change image tag `kubearmor/ubuntu-w-utils:latest`

Reply:
Hi @rksharma95 . Ive made the change. Let me know if you need me to change anything else :)